Altis_Life.Altis/core/actions/fn_getDPMission.sqf

```diff
@@ -24,7 +24,7 @@ if (str(_target) in LIFE_SETTINGS(getArray,"delivery_points")) then {
 life_dp_start = _target;
 life_delivery_in_progress = true;
 
-life_dp_point = call compile format ["%1",_dp];
+life_dp_point = parseSimpleArray format ["%1",_dp];
```

Comment:
We've gotta be careful, not every call compile can become parseSimpleArray. In this instance, there is no array, as once compiled its an object.

Reply:
that could become `missionNamespace getVariable _dp` tho :)

or maybe `missionNamespace getVariable [_dp, objNull]` with isNull check to be safe